### DISCUSSION TITLE
Allow filtering out links that were posted in-game

### DISCUSSION THREAD

**Discussion by Spencer021** [gameserverapp/Platform#609] [Suggestions]
If a player sends a link in the ark chat, it lets it post to the discord, this can cause issues if you have the "chat" channel set so anyone can see it. Maybe prevent links from going to discord? I have a bot but since GSA is technically "admin" it still allows it.
![image](https://github.com/gameserverapp/Platform/assets/58157745/3fdf6e12-2d81-43ea-8170-ff5f5507d0ca)
![image](https://github.com/gameserverapp/Platform/assets/58157745/68bd804b-387c-42f6-a07c-9e497a0e1e0a)

**Comment by mvaessen** [comment]
converted to a suggestion
